pgrx-tests/src/framework.rs

```diff
@@ -673,13 +773,34 @@ fn get_pid_file() -> eyre::Result<PathBuf> {
     Ok(pgdata)
 }
 
+#[inline]
 pub(crate) fn get_pg_dbname() -> &'static str {
     "pgrx_tests"
 }
 
 pub(crate) fn get_pg_user() -> String {
-    std::env::var("USER")
-        .unwrap_or_else(|_| panic!("USER environment var is unset or invalid UTF-8"))
+    get_runas().unwrap_or_else(|| {
+        std::env::var("USER")
+            .unwrap_or_else(|_| panic!("USER environment var is unset or invalid UTF-8"))
+    })
+}
+
+#[inline]
+fn get_runas() -> Option<String> {
+    match std::env::var("CARGO_PGRX_TEST_RUNAS") {
+        Ok(s) => Some(s),
+        Err(e) => match e {
+            VarError::NotPresent => None,
+            VarError::NotUnicode(e) => {
```

Comment:
just noticed a warning. alternately we can print it after doing a `.to_string_lossy()`.
```suggestion
            VarError::NotUnicode(_) => {
```